### PR TITLE
implements technical edits to topics about exported fields and CLI options

### PR DESCRIPTION
**Pull request by dedemorton**
Minor edits for consistency.

### CODE REVIEW

File: docs/fields.asciidoc [hunk 1]
```diff
@@ -69,7 +69,7 @@ type: string
 
 required: True
 
-File the line was read from.
+The file the line was read from.
```
Comment (dedemorton) [inline docs/fields.asciidoc]: @monicasarbu Does this field contain the file name or the file path? If it's the file name, I'll change this to say, "The name of the file from which the line was read."

Comment (monicasarbu) [inline docs/fields.asciidoc]: @dedemorton It's the complete filepath, for example: "/var/log/system.log".

**Comment by dedemorton** [comment]
@monicasarbu I pushed this change into my fork, but realized today that you might have wanted me to push it to master. Can you merge this, or do you want me to do that? thanks!

**Comment by dedemorton** [comment]
@monicasarbu Issue should be fixed now.

**Pull request merged**